nncf/openvino/graph/model_utils.py

```diff
@@ -45,11 +48,27 @@ def insert_null_biases(model: ov.Model, graph: NNCFGraph) -> ov.Model:
     transformation_layout = TransformationLayout()
     model_transformer = ModelTransformerFactory.create(model)
     for node_without_bias in nodes_without_biases:
-        bias_insertion_command = OVCommandCreator.create_command_to_insert_bias(node_without_bias)
+        const_value = create_bias_constant_value(node_without_bias, graph, 0)
+        bias_insertion_command = OVCommandCreator.create_command_to_insert_bias(node_without_bias, const_value)
         transformation_layout.register(bias_insertion_command)
     return model_transformer.transform(transformation_layout)
 
 
+def create_bias_constant_value(node_without_bias: NNCFNode, graph: NNCFGraph, value: Any) -> np.ndarray:
+    """
+    Creates bias value constant array filled by given value.
+
+    :param node_without_bias: NNCFNode to add bias to.
+    :param graph: Target NNCFgraph.
+    :param value: Value to fill bias constant array.
+    :return: Bias value constant array filled by given value.
+    """
+    node_shape = graph.get_output_edges(node_without_bias)[0].tensor_shape
+    bias_shape = [1] * len(node_shape)
+    bias_shape[1] = node_shape[1]
```

Comment:
It may be more flexible to use the `output_channel_axis` attribute from the biased layer metatype. But need to be tested.

Reply:
Did it and have updated tests